client/messaging/connected-username.tsx

```diff
@@ -20,7 +20,7 @@ const Username = styled.span`
  * This component is connected to the store where it tries to find the user. All the services using
  * it should ensure that the user is loaded in the store properly.
  */
-export function ConnectedUsername(props: { userId: SbUserId }) {
+export function ConnectedUsername(props: { userId: SbUserId; isMention?: boolean }) {
```

Comment:
Setting a default value for `isMention` here is preferable to just allowing it to be undefined, imo. It documents the behavior better in the case that a value isn't provided.